src/app/vault/ciphers.component.ts

```diff
@@ -80,6 +86,90 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
         this.didScroll = this.pagedCiphers.length > this.pageSize;
     }
 
+    async loadOranizations(): Promise<Map<string, string>> {
+        const organizations = await this.userService.getAllOrganizations();
+        const organizationNames = new Map<string, string>();
+
+        if (organizations.length > 0) {
+            organizations.forEach(organization => {
+                organizationNames.set(organization.id, organization.name);
+            });
+        }
+
+        return organizationNames;
+    }
+
+    getOrganizationById(c: CipherView): string {
```

Comment:
Possibly rename to getOrganizationNameById and do we need to pass in the whole cipher? If we only pass in the orgId the name would also fit better.

Reply:
Other functions pass the entire cipher so I tried to do the same. But I can change to only pass the necessary information if that's better?